modules/test/playwright/tests/commerce/commerceChannelDefaults.spec.ts

```diff
@@ -341,3 +344,191 @@ test('LPD-26142 A Sales Agent can manage channel defaults', async ({
+	});
+
+	accounts.push(account1);
+
+	apiHelpers.data.push({id: account1.id, type: 'account'});
+
+	const account2 = await apiHelpers.headlessAdminUser.postAccount({
+		name: getRandomString(),
+		type: 'business',
+	});
+
+	accounts.push(account2);
+
+	apiHelpers.data.push({id: account2.id, type: 'account'});
+
+	await apiHelpers.headlessAdminUser.assignAccountToOrganization(
+		account1.id,
+		organization.id
+	);
+
+	await apiHelpers.headlessAdminUser.assignAccountToOrganization(
+		account2.id,
+		organization.id
+	);
+
+	await apiHelpers.headlessAdminUser.assignUserToAccountByEmailAddress(
+		account1.id,
+		[userAccount.emailAddress]
+	);
+
+	await apiHelpers.headlessAdminUser.assignUserToOrganizationByEmailAddress(
+		organization.id,
+		userAccount.emailAddress
+	);
+
+	await apiHelpers.jsonWebServicesUser.assignUsersToSite(
+		site.id,
+		userAccount.id
+	);
+
+	await performLogout(page);
+	await performLogin(page, userAccount.alternateName);
+
+	try {
+		await page.goto(`/web/${site.name}/${layout.friendlyUrlPath}`);
+
+		await expect(
+			accountManagementWidgetPage.accountCell(account1.name)
+		).toBeVisible();
+		await expect(
+			await accountManagementWidgetPage.accountsTableRowLink(
+				account1.name
+			)
+		).toHaveCount(0);
+		await expect(
+			accountManagementWidgetPage.accountCell(account2.name)
+		).toHaveCount(0);
+
+		await performLogout(page);
+		await performLogin(page, 'test');
+
+		const accountManagerRole =
+			await apiHelpers.headlessAdminUser.getRoleByExternalReferenceCode(
+				'Account Manager'
+			);
+
+		await apiHelpers.jsonWebServicesUser.assignUsersToRole(
+			accountManagerRole.id,
+			userAccount.id
+		);
+
+		await performLogout(page);
+		await performLogin(page, userAccount.alternateName);
+
+		for (const account of accounts) {
+			await page.goto(`/web/${site.name}/${layout.friendlyUrlPath}`);
+
+			await (
+				await accountManagementWidgetPage.accountsTableRowLink(
+					account.name
+				)
+			).click();
+
+			await accountsPage.channelDefaultsTab.click();
+
+			const channelEntryHeaderNames = [
+				'Billing Addresses',
+				'Channel Account Managers',
+				'Currencies',
+				'Delivery Terms and Conditions',
+				'Discounts',
+				'Payment Methods',
+				'Payment Terms and Conditions',
+				'Price Lists',
+				'Shipping Addresses',
+				'Shipping Options',
+			];
+
+			for (const channelEntryHeaderName of channelEntryHeaderNames) {
+				commerceChannelDefaultsPage.channelEntryHeader(
+					channelEntryHeaderName
+				);
+			}
+
+			const channelEntryNames = [
+				'Billing',
+				'Currencies',
+				'Delivery',
+				'Discounts',
+				'PaymentCommerceTerm',
+				'Price',
+				'ShippingCommerceAddress',
+				'Users',
+			];
+
+			for (const channelEntryName of channelEntryNames) {
+				const channelEntry =
+					commerceChannelDefaultsPage.channelEntry(channelEntryName);
+				const addButton =
+					commerceChannelDefaultsPage.channelEntryAddButton(
+						channelEntryName
+					);
+
```

Comment:
can we inline this instead of storing in the variables? Are used only in the expect